src/ros_filter.cpp

```diff
@@ -46,6 +48,26 @@
 
 namespace RobotLocalization
 {
+
+  //! @brief Compute covariance volume
```

Comment:
Done/Moved to `filter_utilities.h`

I'll squash the commit I've just pushed before merging.